.github/workflows/ci-idris2.yml

```diff
@@ -300,8 +300,8 @@ jobs:
           echo "deb http://security.ubuntu.com/ubuntu hirsute universe" | sudo tee -a /etc/apt/sources.list
           sudo apt-get update
           sudo apt-get install -y -t hirsute chezscheme
-          echo "$HOME/.idris2/bin" >> $GITHUB_PATH
-          chmod +x $HOME/.idris2/bin/idris2 $HOME/.idris2/bin/idris2_app/*
+          echo "$HOME/.idris2/bin" >> "$GITHUB_PATH"
+          chmod +x "$HOME/.idris2/bin/idris2" "$HOME/.idris2/bin/idris2_app/"*
```

Comment:
I'm not at the computer to verify right now but does this glob outside the quotes work as intended? Looks a bit off but I might have just never seen the asterisk put after the quotes before.

Reply:
i added the `false` to disable a one last racket bootstrap run, sorry for not commenting on that! IIRC racket was previously disabled in the CI because it takes too long and it just times out.

the new quotes are because the new linter version is smart enough to understand that those are pieces of shell scripts, and it complains that the ENV vars may contain spaces and/or globs.

how/where is the quoting inconsistent? note that i left the star out of the quotes, so that it is interpreted as an anything-marker.

Reply:
I don't think we should disable the windows racket check prematurely. As far as I know it has been working consistently for quite some time and it seems to usually complete in 15 minutes. 

> how/where is the quoting inconsistent?

I was reviewing without my laptop to hopefully get this merged right away because it looks like the right solve for failing linting but I wasn't able to verify all of the quotation changes acted the same way (and I'm not a bash expert). The inconsistently is around what exactly gets quoted. Sometimes just the variable (which is probably the only thing that needs to be quoted for the linter to be happy), sometimes it's the variable and the whole path up to the asterisk, sometimes the last forward slash is outside the quotes.

Reply:
i see, thanks!

ok, i've updated the PR: eliminated the racket thing, and made the quotes uniform.

let me know if there's anything else.